index.html

```diff
@@ -0,0 +1,136 @@
+<html lang="en">
+<head> 
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
consider giving your page title descriptive name. for better search engine listing.